### PR TITLE
Null dereference check on EventEnvelopePayload toJson utility

### PR DESCRIPTION
**Pull request by abhishekjain88**
We were seeing null pointer exceptions on the export DB internally on Wasabi's intuit extension which were caused by unsafe dereferencing of null objects while using the toJson utility on the event envelope. This change addresses the fix.

There is a parallel PR on wasabi intuit as well.

### CODE REVIEW

**Comment by iizrailevsky** [comment]
Just few tidy comments. Once resolved and integration tests pass -> :shipit:

File: modules/event-objects/src/main/java/com/intuit/wasabi/eventobjects/EventEnvelopePayload.java [hunk 1]
```diff
@@ -88,18 +88,25 @@ public String toJson() {
...
-        eventJson.put("eventPayload", event.getPayload());
-        eventJson.put("value", event.getValue());
+        eventJson.put("experimentID",
+                assignment != null && assignment.getExperimentID() != null ?
+                        assignment.getExperimentID().toString(): null);
+        eventJson.put("context",
+                assignment != null && assignment.getContext() != null ? assignment.getContext().toString(): null);
+        eventJson.put("epochTimestamp",
+                event != null && event.getTimestamp() != null ? event.getTimestamp().getTime(): null);
+        eventJson.put("eventType", event != null && event.getType() != null ? event.getType() + "": "null");
```
Comment (bargenilesh) [inline modules/event-objects/src/main/java/com/intuit/wasabi/eventobjects/EventEnvelopePayload.java]: Minor: You probably don't need "&& event.getType() != null" & "&& event.getName() != null"

**Comment by bargenilesh** [comment]
Except a very minor comment, looks good... Provided integration tests pass :shipit:

File: modules/event-objects/src/main/java/com/intuit/wasabi/eventobjects/EventEnvelopePayload.java (hunk 1 above)
Comment (abhishekjain88) [inline modules/event-objects/src/main/java/com/intuit/wasabi/eventobjects/EventEnvelopePayload.java]: Thanks @bargenilesh for pointing that out. I didn't realize that you could concatenate with a "null" string.

Comment (abhishekjain88) [inline modules/event-objects/src/main/java/com/intuit/wasabi/eventobjects/EventEnvelopePayload.java]: Taken care in latest commit!

**Pull request merged**